docs/user/azure/install_upi.md

```diff
@@ -209,7 +209,7 @@ export VHD_URL=`curl -s https://raw.githubusercontent.com/openshift/installer/ma
 Copy the chosen VHD to a blob:
 
 ```sh
-az storage container create --name vhd --account-name ${CLUSTER_NAME}sa
+az storage container create --name vhd --account-name ${CLUSTER_NAME}sa --account-key ${ACCOUNT_KEY}
```

Comment:
Per our QE testing result, I think we do not need "--account-key" option on this step, right?

Reply:
az storage container create --name vhd --account-name ${CLUSTER_NAME}sa
There are no credentials provided in your command and environment, we will query for account key for your storage account.
It is recommended to provide --connection-string, --account-key or --sas-token in your command as credentials.

Reply:
It is just a warning, when no `--account-key` is provided, it will default to use the az login's service principal to access. And QE's original script was working well, also prove this command does not need `--account-key` as a MUST. If we have no specific complaint about that, I think it is better to not touch with it, only focus on `--public-access blob` issue.